SU-SWS/stanford_profile
# D8CORE-1873: Wrapping bug for the multi-row

# READY FOR REVIEW

# Summary
- This is to fix wrapping elements

# Needed By (Date)
- Tuesday

# Urgency
- Critcal

# Steps to Test

1. Pull this branch. Make sure you have the updates to have multi-row.
2. Generate some content. (lando drush spstc)
3. Verify that the media elements at 4 across are staying 4 across.

# Affected Projects or Products
- Needs the multi per row.

# Associated Issues and/or People
-  D8CORE-1873

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
